opflexagent/test/test_gbp_vpp_agent.py

```diff
@@ -13,13 +13,15 @@
 import shutil
 import sys
 
+from contextlib2 import contextmanager
+from contextlib2 import ExitStack
 import mock
 sys.modules["apicapi"] = mock.Mock()
 sys.modules["pyinotify"] = mock.Mock()
 sys.modules['opflexagent.vpplib'] = mock.MagicMock()
 sys.modules['opflexagent.vpplib.VPPApi'] = mock.MagicMock()
 
-import contextlib
+#import contextlib
```

Comment:
We should just remove this line completely I think.

Reply:
Oh, I might've missed it. I will update it in the next commit.